UpComingMovies/app/src/main/res/layout/activity_main.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context="com.example.douglas.upcomingmovies.MainActivity">
+
+    <android.support.v7.widget.RecyclerView
+        android:id="@+id/rv_movieservice"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"/>
+
+    <TextView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:id="@+id/tv_error_message_display"
+            android:padding="16dp"
+            android:text="@string/error_message"
+            android:textSize="20sp"

```

Comment:
Do you see any problem here by using the sp unit?

Reply:
So I did use SP(AKA for Scale-independent Pixels) because it respects the user preference. For example, if you want everyone to be able to see the texts in your app, even people with vision disabilities, you should use SP, because these people that use a different scale on their phones can make use of your app too :D